### PR TITLE
Added validation to players social links

### PR DESCRIPTION
**Pull request by Morabotti**
Implementation of #73

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -25,22 +25,16 @@
...
     }
   ],
   "browser_action": {
     "default_icon": "icon48.png",
     "default_popup": "popup.html"
   },
   "background": {
-    "scripts": [
-      "background.js"
-    ],
```
Comment (timche) [inline src/manifest.json:43]: Could you please revert these changes? Thanks

Comment (Morabotti) [reply src/manifest.json]: Seems like `yarn prettier` formats also json files. My dev environment could also be misconfigured.
